content/en/getting_started/agent/_index.md

```diff
@@ -12,90 +12,219 @@ further_reading:
+  Pid: 9801
+  Go Version: go1.17.6
+  Python Version: 3.8.11
+  Build arch: amd64
+  Agent flavor: agent
+  Check Runners: 6
+  Log Level: info
+```
 
-#### Agent
+#### Events
 
-The metrics below are available with Agent v6. For Agent v5, see the [Agent Metrics][18] integration.
+In the Datadog UI, go to the Events Explorer Page **Events > Explorer**. The Agent sends events to Datadog when an Agent is started or restarted. The following message will display if your Agent successfully installs:
 
-| Metric                           | Description                                                                                                          |
-| -------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
-| **datadog.agent.python.version** | Shows a value of `1` if the Agent is reporting to Datadog. The metric is tagged with the `python_version`. |
-| **datadog.agent.running**        | Shows a value of `1` if the Agent is reporting to Datadog.                                                 |
-| **datadog.agent.started**        | A count sent with a value of `1` when the Agent starts (available in v6.12+).                                        |
+```text
+Datadog agent (v. 7.XX.X) started on <Hostname>
+```
 
-#### Checks
+#### Service Checks
 
-Depending on your platform, the Agent has several core checks enabled by default that collect metrics.
+The Agent is set up to provide the following service checks:
 
-| Check       | Metrics       | Platforms          |
-| ----------- | ------------- | ------------------ |
-| CPU         | [System][19]  | All                |
-| Disk        | [Disk][20]    | All                |
-| Docker      | [Docker][21]  | Docker             |
-| File Handle | [System][19]  | All except Mac     |
-| IO          | [System][19]  | All                |
-| Load        | [System][19]  | All except Windows |
-| Memory      | [System][19]  | All                |
-| Network     | [Network][22] | All                |
-| NTP         | [NTP][23]     | All                |
-| Uptime      | [System][19]  | All                |
-| Winproc     | [System][19]  | Windows            |
+  - **datadog.agent.up**:
+    Returns `OK` if the Agent is able to connect to Datadog.
+
+  - **datadog.agent.check_status**:
+    Returns `CRITICAL` if an Agent check is unable to send metrics to Datadog, otherwise returns `OK`.
+
+These checks can be used in the Datadog Platform to visualize the Agent status through Monitors and Dashboards at a quick glance. See [Service Check Overview][20] to learn more.
+
+#### Metrics
+
+In the Datadog UI, go to the Metrics Summary page **Metrics > Summary** and search for the metric **datadog.agent.started** or the metric **datadog.agent.running**. If these metrics are not visible right away, it may take a few minutes for the Agent to send the data to the Datadog Platform.
```

Comment:
```suggestion
In the Datadog UI, go to the Metrics Summary page **Metrics > Summary** and search for the metric `datadog.agent.started` or the metric `datadog.agent.running`. If these metrics are not visible right away, it may take a few minutes for the Agent to send the data to the Datadog Platform.
```